src/jit/morph.cpp

```diff
@@ -3185,7 +3185,16 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* call)
         }
 
 #ifdef _TARGET_ARM_
-
+#ifndef LEGACY_BACKEND
+        if (reMorphing)
```

Comment:
Instead of adding any code, it looks like you can just adjust the `#ifdefs` to share arm64 code. That is, change the above to:
```
#if defined(_TARGET_ARM_) && defined(LEGACY_BACKEND)
```
and below from:
```
#elif defined(_TARGET_ARM64_)
```
to:
```
#elif defined(_TARGET_ARMARCH_)
```